contrib/action_nitro/Makefile

```diff
@@ -1,3 +1,6 @@
+.SUFFIXES:
+SHELL = /bin/sh
```

Comment:
* Some variants of `make` do not set the shell as expected.
* Most people do not know that GNU Make always sets the shell to `/bin/sh`. So, they may have hard time to discover they can not add a BASH/FISH/KSH command to your Makefile (« Why the command I type in my console does not work in the Makefile? »). Here, we clearly state that the shell is always `/bin/sh`.